config/katello.migrations/202011084145-dont-user-pulpcore-deb-on-upgrades.rb

```diff
@@ -0,0 +1,11 @@
+KATELLO = 'katello'.freeze
+USE_PULP_2_FOR_DEB = 'use_pulp_2_for_deb'.freeze
+FOREMAN_PROXY_CONTENT = 'foreman_proxy_content'.freeze
+ENABLE_DEB = 'enable_deb'.freeze
+
+def upgrade
+  answers[KATELLO] = {} unless answers[KATELLO].is_a?(Hash)
+  answers[KATELLO][USE_PULP_2_FOR_DEB] = true
+end
+
+upgrade if answers[KATELLO] && answers[KATELLO][USE_PULP_2_FOR_DEB].nil? && answers[FOREMAN_PROXY_CONTENT][ENABLE_DEB]
```

Comment:
Why is it using `enable_deb` on `foreman_proxy_content` while `katello` also has the same parameter? Also, a separate method is not really needed. I'd also be fine with the simpler:

```suggestion
if answers[KATELLO].is_a?(Hash) && answers[KATELLO][USE_PULP_2_FOR_DEB].nil? && answers[KATELLO][ENABLE_DEB]
  answers[KATELLO][USE_PULP_2_FOR_DEB] = true
end
```

The same is true for the other migrations.

Reply:
i just mimicked what was done for the other migrations, should i update those too?

Reply:
I'm fine if you don't. Normally I don't like changing migrations unless there's bugs in them. However, I do like to keep new migrations short.

Reply:
thanks, updated!